docs/configuration.md

```diff
@@ -172,6 +173,18 @@ hub:
   consecutiveFailureLimit: 0
 ```
 
+<a name="increase-http-timeout"></a>
+### Increase http_timeout
+
+[`c.KubeSpawner.http_timeout`](https://jupyterhub.readthedocs.io/en/stable/api/spawner.html#jupyterhub.spawner.Spawner.http_timeout)
+defaults to 30 seconds. During scale and load testing we have seen that sometimes
+we can hit this timeout and the hub will delete the server pod but if we had just waited a few seconds more it
+would have been enough. So if you have node capacity so that pods are being created, but maybe they are just
+slow to come up and are hitting this timeout, you might want to increase it to something like 60 seconds. This
+also seems to vary depending on whether you are using `notebook` or `jupyterlab` / `jupyter-server`, the type of
+backing storage for the user pods (PVCs or shared [object] storage file system), and how many and what kinds of
```

Comment:
We don't need to mention PVCs here as they're just an abstraction used by Kubernetes (not an either-or type of thing, we use PVCs with our s3fs setup).  I think really this just needs to call out s3fs in particular as being slow.

Reply:
```suggestion
backing storage for the user pods (i.e. s3fs shared object storage is known to be slow(er)), and how many and what kinds of
```

How about this wording?